src/test/java/uk/gov/pay/directdebit/infra/PostgresContainer.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.pay.directdebit.infra;
+
+import com.google.common.base.Stopwatch;
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.LogStream;
+import com.spotify.docker.client.exceptions.DockerException;
+import com.spotify.docker.client.messages.ContainerConfig;
+import com.spotify.docker.client.messages.ContainerInfo;
+import com.spotify.docker.client.messages.HostConfig;
+import com.spotify.docker.client.messages.LogConfig;
+import com.spotify.docker.client.messages.PortBinding;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.Integer.parseInt;
+import static java.util.stream.Collectors.joining;
+
+public class PostgresContainer {
```

Comment:
Great idea! 👍

Reply:
Is actually copied from somewhere! adminusers or connector